src/main/java/Duke.java

```diff
@@ -1,10 +1,239 @@
+        System.out.println("\t- adds specified task to the list" + "\n");
+        System.out.println("\tdeadline | <task>  | /by | <deadline>");
+        System.out.println("\t- adds specified task and deadline and to the list" + "\n");
+        System.out.println("\tevent    | <task>  | /at | <timing>");
+        System.out.println("\t- adds specified task and timing to the list" + "\n");
+        System.out.println("\tdone     | <index>");
+        System.out.println("\t- marks existing task matching the specified index"
+                + "as completed in the list" + "\n");
+        System.out.println("<> indicates an input field and | is a field separator.");
+        System.out.println(BORDER + NEWLINE);
+    }
+
+    /** Methods that print part of or full list */
+    public static void echo() {
+        System.out.println(BORDER);
+        System.out.println("New task added: ");
+        System.out.print("\t");
+        tasks[tasksCount-1].printTask();
+        System.out.print("\n");
+//        System.out.println("There are " + tasksCount + " tasks in your list.");
+        System.out.print("There ");
+        System.out.print(tasksCount > 1 ? "are " : "is ");
+        System.out.print(tasksCount);
+        System.out.print(tasksCount > 1 ? " tasks" : " task");
+        System.out.println(" in your list.");
+        System.out.println(BORDER + NEWLINE);
+    }
+
+    public static void printList() {
+        System.out.println(BORDER);
+        System.out.println("Here are the tasks in your list:");
+        for (int i=0; i<tasksCount; i++) {
+            System.out.print("\t" + tasks[i].getIndex() + ". ");
+            tasks[i].printTask();
+            System.out.print("\n");
+        }
+        System.out.println(BORDER + NEWLINE);
+    }
+
+
+    /** Methods that add or modify a task in the list */
+    public static void addTodo(String description) {
+        tasks[tasksCount] = new Task(description, tasksCount+1);
+        tasks[tasksCount].setStatus(DEFAULT_STATUS);
+        tasks[tasksCount].setType("T");
+        tasksCount++;
+        echo();
+    }
+
+    public static void addDeadline(String description, String deadline) {
+        tasks[tasksCount] = new Deadline(description, tasksCount+1, deadline);
+        tasks[tasksCount].setStatus(DEFAULT_STATUS);
+        tasks[tasksCount].setType("D");
+        tasksCount++;
+        echo();
+    }
+
+    public static void addEvent(String description, String time) {
+        tasks[tasksCount] = new Event(description, tasksCount+1, time);
+        tasks[tasksCount].setStatus(DEFAULT_STATUS);
+        tasks[tasksCount].setType("E");
+        tasksCount++;
+        echo();
+    }
+
+    public static void markInList(int id) {
+        tasks[id-1].setStatus(DONE_STATUS);
+        System.out.println(BORDER);
+        System.out.println("Nice! This task is now done:");
+        System.out.print("\t");
+        tasks[id-1].printTask();
+        System.out.print("\n");
```

Comment:
You have already declared Newline so you can use it here

Reply:
You're right, thank you! Will do.